web/chromecom.js

```diff
@@ -81,10 +86,7 @@
         var streamUrl = response.streamUrl;
         if (streamUrl) {
           console.log('Found data stream for ' + file);
-          PDFViewerApplication.open(streamUrl, {
-            length: response.contentLength
-          });
-          PDFViewerApplication.setTitleUsingUrl(file);
+          callback(streamUrl, response.contentLength, file);
```

Comment:
The callback received three parameters here, but at line 369 only two parameters are used.